src/d3a/models/area/__init__.py

```diff
@@ -62,7 +64,8 @@ def __init__(self, name: str = None, children: List["Area"] = None,
                  balancing_spot_trade_ratio=ConstSettings.BalancingSettings.SPOT_TRADE_RATIO,
                  event_list=[],
                  transfer_fee_pct: float = None,
-                 transfer_fee_const: float = None):
+                 transfer_fee_const: float = None,
+                 external_connection_available=False):
```

Comment:
Can you add any integration/unittest showcasing how external agents could be connected using this flag?

Reply:
The tricky part is that real integration tests actually require Redis for that. I could add a small script though that exhibits the behavior we want, though I do not know how helpful it will be if  it does not run on Travis. Let me try adding unit tests, maybe these will be enough.

Reply:
That would be great, if its too much of a trouble, could be skipped for later.